docs/how-tos/use-multi-format-reader.md

```diff
@@ -0,0 +1,399 @@
+# How to use the built-in MultiFormatReader
+While building on the ```BaseReader``` allows for the most flexibility, in most cases it is desirable to implement a reader that can read in multiple file formats and then populate the template based on the read data. For this purpose, `pynxtools` has the [**`MultiFormatReader`**](https://github.com/FAIRmat-NFDI/pynxtools/blob/master/src/pynxtools/dataconverter/readers/multi/reader.py), which can be readily extended for your own data. In this how-to guide, we will focus on an implementation using a concrete example. If you are also interested in the general structure of the `MultiFormatReader`, you can find more information [here](../learn/multi-format-reader.md).
+
+## Getting started
+
+**Note: You can find all of the data and the developed python scripts [here](https://download-directory.github.io/?url=https://github.com/FAIRmat-NFDI/pynxtools/tree/master/examples/mock-data-reader).**

```

Comment:
This is expected, since the path does not exist in the master branch yet (will only be there after merging). I will check again after merging that the download actually works.